ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -831,7 +831,7 @@ @implementation ORKFormItemScaleCell {
 }
 
 - (id<ORKScaleAnswerFormatProvider>)formatProvider {
-    if(_formatProvider == nil){
+    if (_formatProvider == nil){
         _formatProvider = (id<ORKScaleAnswerFormatProvider>)[self.formItem.answerFormat impliedAnswerFormat];
```

Comment:
No I was referring to the `if` statement. It should look like `if (_formatProvider == nil) {`